tests/playwright-test/to-have-screenshot.spec.ts

```diff
@@ -196,36 +178,6 @@ test('should ignore non-documented options in toHaveScreenshot config', async ({
   expect(pngComparator(fs.readFileSync(snapshotOutputPath), whiteImage)).toBe(null);
 });
 
-test('screenshotPath should include platform and project name by default', async ({ runInlineTest }, testInfo) => {
```

Comment:
Why remove this one? It looks like the only test that covers our default behavior today?

Reply:
This is for `screenshotPath`. The `snapshotPathTemplate` is covered with tests separately